packages/core/src/branding/get-branding-css.ts

```diff
@@ -0,0 +1,119 @@
+/**
+ * Copyright (c) 2024, WSO2 LLC. (https://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import {BrandingPreferenceThemeInterface} from '../models/branding-api-response';
+import isEmpty from '../utils/is-empty';
+
+const getBrandingCSS = (theme: BrandingPreferenceThemeInterface): string => {
+  if (!theme) {
+    return '';
+  }
+
+  const footerFontColor: string = !isEmpty(theme[theme.activeTheme].footer.font.color)
```

Comment:
Lets create a local variable for theme[theme.activeTheme] and reference that everywhere.

Reply:
I took this from a current implementation in identity apps. Its better to update.
Added with 97b0093868651a2936065710b99d93c0a21e02a0